lib/hanami/commands/server.rb

```diff
@@ -24,12 +27,15 @@ class Server < ::Rack::Server
       # @since 0.1.0
       # @see Hanami::Environment#initialize
       def initialize(options)
+        @entr_options =  options.merge(code_reloading: false).inject([]) {|res, (k, v)| res << "--#{k}=#{v}" ; res}.join(" ")
```

Comment:
@aderyabin Does this means we're open to accept any kind of CLI args and pass down to `entr`?